notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb

```diff
@@ -0,0 +1,2444 @@
+{
```

Comment:
<p>ok, the pipeline finished. Example is hanging. Need some code/text cells that check that the pipeline did something, like create the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>